__tests__/app.test.js

```diff
@@ -175,4 +175,98 @@ describe('APP', () => {
             })
         });
     })
+    describe('POST /api/articles/:article_id/comments', () => {
+        test('should insert a new comment and return a 201 status, along with an object of the new comment', () => {
+            const newComment = {
+                author: 'icellusedkars',
+                body: 'We will see about that!'
+            }
+            return request(app)
+            .post('/api/articles/13/comments')
+            .send(newComment)
+            .expect(201)
+            .then((response) => {
+                const expectedResponse = {
+                    body: 'We will see about that!',
+                    article_id: 13,
+                    author: 'icellusedkars',
+                }
```

Comment:
There should be more keys than this in the response. Please console.log to check and then add them to your test :)